prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir

```diff
@@ -1,14 +1,15 @@
 {
   "resourceType" : "Bundle",
-  "id" : "1673379887653974000.2d3730ba-0420-4ead-a33c-878fcecf5941",
+  "id" : "1674153054109185000.9fae9996-785b-428d-a4f8-5d6d0de32cc6",
   "meta" : {
-    "lastUpdated" : "2023-01-10T13:44:47.663-06:00"
+    "lastUpdated" : "2023-01-19T13:30:54.124-05:00"
   },
   "identifier" : {
+    "system" : "https://reportstream.cdc.gov/prime-router",
```

Comment:
This is the new field

Reply:
Is this not checked in the tests? I'm just unclear of why this wasn't needed in other `.fhir` files for the tests to pass.